libraries/AP_RCProtocol/AP_RCProtocol.h

```diff
@@ -21,6 +21,10 @@
 #define MAX_RCIN_CHANNELS 18
 #define MIN_RCIN_CHANNELS  5
 
+#ifndef AP_RCPROTOCOL_FASTSBUS_ENABLED
+#define AP_RCPROTOCOL_FASTSBUS_ENABLED 1
```

Comment:
!IOMCU? - Not sure what the decision was here

Reply:
Yep, that was done